uint8 action
uint8 USE_POSITION = 1
uint8 USE_VELOCITY = 2
uint8 USE_ACCELERATION = 3
uint8 USE_MIXED = 4

float64 x
float64 y
float64 z
float64 vx
float64 vy
float64 vz
float64 ax
float64 ay
float64 az

float64 yaw
float64 time
string frame

bool reset
